egs2/dns_icassp22/enh1/run.sh

```diff
@@ -26,4 +26,5 @@ test_sets="cv_synthetic tt_synthetic"
     --use_noise_ref false \
     --max_wav_duration 31 \
     --inference_model "valid.loss.best.pth" \
+    --nj 8 \
```

Comment:
Perhaps you could delete this line.

Reply:
I added this line since it's much easier to run into OOM error with the default `nj=32` when we're working with 48 kHz.

Reply:
I mean in the preprocessing stages